core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -189,16 +185,13 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
     var _getImageDimensionsInState = function(state) {
       var fileDimensions = {};
       if (state.interaction.id === INTERACTION_TYPE_IMAGE_CLICK_INPUT) {
-        var filepathObject = (
+        var filename = (
```

Comment:
Seems like this should be "filepath" because it reads off of "imagePath"

Reply:
Same as above.

Reply:
Done!